src/components/Badge/__snapshots__/Badge.test.ts.snap

```diff
@@ -0,0 +1,113 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`<Badge /> Default story renders snapshot 1`] = `
+<span
+  class="badge__wrapper"
+>
+  <div
+    class="badge badge--empty"
+  />
```

Comment:
This is going to need some sort of screen reader text 🤔

Reply:
Added associated `aria-label`